tests/test_load_imaged.py

```diff
@@ -63,10 +63,12 @@ def test_channel_dim(self):
         test_image = np.random.rand(*spatial_size)
         with tempfile.TemporaryDirectory() as tempdir:
             filename = os.path.join(tempdir, "test_image.nii.gz")
-            nib.save(nib.Nifti1Image(test_image, affine=np.eye(4)), filename)
+            itk_np_view = itk.image_view_from_array(test_image.T.copy())
```

Comment:
shall we keep the pervious implementation, it's a valid test case.

Reply:
Sure, restored the test in the latest commit.

Thanks.